### ISSUE TITLE
Content must be inside of landmark elements

### ISSUE THREAD

**Issue by gldgrnt** [wmcadigital/wmn-design-system#280]
**Describe the bug**
AXE is flagging the following content as being outside of landmark elements:

- Beta banner
- Breadcrumbs
- Feedback loop banner

**Expected behaviour**
Content should be within the `<header>`, `<main>` or `<footer>` tags

**Comment by daylesalmon** [comment]
Relates to #287
